src/Content/Migration/Base.php

```diff
@@ -192,10 +192,10 @@ public function getMWDBO()
 	 **/
 	private function getRootCredentials()
 	{
-		$secrets   = DS . 'etc'  . DS . 'hubzero.secrets';
-		$conf_file = DS . 'root' . DS . '.my.cnf';
-		$hub_maint = DS . 'etc'  . DS . 'mysql' . DS . 'hubmaint.cnf';
-		$deb_maint = DS . 'etc'  . DS . 'mysql' . DS . 'debian.cnf';
+		$secrets   = DIRECTORY_SEPARATOR . 'etc'  . DIRECTORY_SEPARATOR . 'hubzero.secrets';
```

Comment:
Do we still need directory separator constants, or is now not quite the time to remove them all?

Reply:
We don't really need them, I guess. I was just trying to get rid of usage of CMS-specific constants.